packages/terra-date-picker/tests/wdio/calendar-filter-spec.js

```diff
@@ -0,0 +1,93 @@
+Terra.describeViewports('Calendar Filter', ['tiny'], () => {
+  describe('Calendar-Filter-Default', () => {
+    before(() => browser.url('/#/raw/tests/terra-date-picker/date-picker/calendar-filter/calendar-filter-default'));
+
+    Terra.it.validatesElement();
+  });
+
+  describe('Calendar-Filter with Default Date Excluded', () => {
+    before(() => browser.url('/#/raw/tests/terra-date-picker/date-picker/calendar-filter/calendar-filter-default-date-excluded'));
+
+    Terra.it.validatesElement();
+  });
+
+  describe('Calendar-Filter with Default Date Out Of Range', () => {
+    before(() => browser.url('/#/raw/tests/terra-date-picker/date-picker/calendar-filter/calendar-filter-default-date-out-of-range'));
+
+    Terra.it.validatesElement();
+  });
```

Comment:
Not a big deal, but we should start writing wdio test like:

```suggestion
  describe('Calendar-Filter with Default Date Out Of Range', () => {
    it('renders as expected', () => {
        browser.url('/#/raw/tests/terra-date-picker/date-picker/calendar-filter/calendar-filter-default-date-out-of-range'));
        Terra.validates.element();
    });
  });
```

Reply:
updated : [940b5b3363f](https://github.com/cerner/terra-framework/pull/1134/commits/8ec0355f75cba2e79d27fbd326b343bec97afba4#diff-dfc44135d5b407abfc777940b5b3363f)